products/statement-generator/src/components/PrimaryLandingSection.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from 'react';
+import { Theme, makeStyles, createStyles } from '@material-ui/core';
+
+import Button from 'components/Button';
+import ContentContainer from 'page-layout/ContentContainer';
+
+interface ComponentProps {
+  goNextPage: () => void;
+}
+
+interface StyleProps {
+    theme?: string;
+}
+
+const useMediaQuery = (query: string) => {
+    const [matches, setMatches] = useState(false);
+
+    useEffect(() => {
+        const media = window.matchMedia(query);
+        if(media.matches !== matches) {
+            setMatches(media.matches);
+        }
+        const listener = () => setMatches(media.matches);
+        window.addEventListener("resize", listener);
+        return () => window.removeEventListener("resize", listener);
+    }, [matches, query]);
+
+    return matches;
+};
```

Comment:
Clever! Instead of adding an additional listeners what you can do instead is have two images and use css to `display: none` the vertical image when not in mobile breakpoints and then `display: none` the side image when not in desktop breakpoints.